src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -113,6 +123,14 @@ protected OutgoingNode createHelperNode(int argsLength) {
             return new CallOutgoingNode(argsLength);
         } else if (name.equals("new")) {
             return new NewOutgoingNode(argsLength);
+        } else if (name.equals("to_a") || name.equals("to_ary")) {
+            return new ToAOutgoingNode();
+        } else if (name.equals("respond_to?")) {
+            return new RespondToOutgoingNode();
+        } else if (name.equals("inspect")) {
+            return new InspectOutgoingNode();
+        } else if (name.equals("send")) {
```

Comment:
`__send__` instead of `send` might make sense - but lots of the other special forms could also be message names. If you get a conflict, use `Truffle::Interop.invoke`.